src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -201,6 +201,20 @@ You can pass many different options to tracing, but it comes with reasonable def
 - If available: Browser Resources (Scripts, CSS, Images ...)
 - If available: Browser Performance API Marks
 
+*tracingOrigins Option*
+ 
+The default value of `tracingOrigins` is : `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//]})`. This options will attach the `sentry-trace` header to all outgoing XHR/fetch requests that contain a string in the list or match the regex. In case your frontend is making requests to a different domain you might want to add it there so we attach the `sentry-trace`.
+
+*Example:*
+
+- Your frontend is running `mywebapp.com`
+- Your backend is running `api.mybackend.com`
+- You setup your frontend to make API calls to `api.mybackend.com`
+- Then the option need to be configured like this `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//, 'api.mybackend.com']})`
```

Comment:
```suggestion
- Then the option need to be configured like this `new ApmIntegrations.Tracing({tracingOrigins: ['api.example.com']})`
```

Reply:
Do we have a similar configuration in Python, or do we just send the header regardless of destination domain? cc @untitaker

Reply:
We do it regardless of destination domain and do not let the user configure this, but I think we will not get away with either in the long-term